client/my-sites/checkout/src/components/wp-order-review-line-items.tsx

```diff
@@ -398,6 +399,15 @@ function LineItemWrapper( {
 			return true;
 		}
 
+		if (
```

Comment:
This looks like a duplicate of lines 411-477 with the addition of isStreamlinedPriceDropdownTreatment check.

Reply:
Thanks for catching that!
I replaced it with a  single`isStreamlinedPriceCheckoutTreatment` outside the hooks callback. It'll be useful for the rest of the changes in the Order list.